src/pages/iou/steps/IOUAmountPage.js

```diff
@@ -42,11 +42,17 @@ const propTypes = {
 
         /** Selected Currency Code of the current IOU */
         selectedCurrencyCode: PropTypes.string,
-    }).isRequired,
+    }),
 
     ...withLocalizePropTypes,
 };
 
+const defaultProps = {
+    iou: {
+        loading: false,
```

Comment:
unused `loading` prop, removed it here https://github.com/Expensify/App/pull/2723